### DISCUSSION TITLE
Open Windows Size and Positions

### DISCUSSION THREAD

**Discussion by KPreston2900050** [Open-Shell/Open-Shell-Menu#1549] [Ideas]
When I did research this Microsoft said no one ever complained about them removing it from Windows 7
Then you find out that thousands of users complained and it was ignored.

XP had an issue with BAGS.MRU so instead of fixing the problem they removed the feature altogether.
You can get Shell Folder Fix which does work with Windows 11 but is not solid and no longer updated.

Which brings me to Open Shell because you seem to at least hear users unlike Start is Back which needs an account and go through a hassle just to ask a question.

I hack the crap out of Windows 11 before I use it.
I have to work with hundreds of files on almost a daily basis and I need to have a folder stay where I opened it and the size I made it..
I do not care to use one folder to handle all of my data transfers.  I do not use that useless ribbon either.

Shell Folder Fix does not work well with multiple monitors and for a while there Windows 11 would lock up and reset explorer when it had a conflict with SFF which would delete the entire database of saved windows position.

So, now that the long winded explanation is over is there any way to incorporate this feature but in a modern way?

Shell Folder Fix is the only alternative that I know of and it is several years old now.

KP

**Comment by pchris7489** [comment]
I didn't know about "Shell Folder Fix" and it would have save me a lot of time and hassle if I knew it earlier. But for those type of problems I heavily used "Autohotkey" to fix the issue. You can very easily create a mini script (hundreds of tutorials out here with a huge community even on discord) to move the windows automatically. Works wonders, obviously, its not as foward as "Shell Folder Fix" with a pre-made GUI but I am sure that some guy already wrote a script on AHK forum to solve your problem.
